src/polyglot-notebooks-vscode-common/src/notebookSerializers.ts

```diff
@@ -72,8 +72,17 @@ export function createAndRegisterNotebookSerializers(context: vscode.ExtensionCo
         };
 
         const notebookoptions: vscode.NotebookDocumentContentOptions = notebookType === commandsAndEvents.DocumentSerializationType.Dib
-            ? { transientOutputs: true, transientDocumentMetadata: { custom: true }, transientCellMetadata: { custom: true } }
-            : {};
+            // FIX: does this solve incorrect dirty state?
+            ? {
```

Comment:
Weird, I'm not seeing any difference between these two conditions. Is this temporary?

Reply:
They're the same. This was an attempt to see if this code does anything at all or is implicated in a bug I'm investigating.